### PR TITLE
Modified yr,mn,dd based on transacton_date

### PR DESCRIPTION
**Pull request by AnushaVedagiri**

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -84,22 +84,22 @@
       <screen x="0" y="0" width="1536" height="824" />
```
Comment (corymickelson) [inline .idea/workspace.xml]: remove .idea/* from github

File: EquiFaxResponse_Bronze/src/main/scala/com/loanpal/etl/equifaxresponse/equifaxresponse_bronze/Main.scala [hunk 2]
```diff
@@ -34,9 +34,15 @@ object Main {
...
-      .withColumn("day", dayofmonth(to_date(interrogateDate(input_file_name), "yyyy-MM-dd")).cast(IntegerType))
+      .withColumn("converted_transactionDate",
+        to_date($"applicants.primaryConsumer.transUnionUSConsumerCreditReport.transUnionUSConsumerCreditReport.transactionDate", "yyyy-MM-dd"))
+      .withColumn("year", when($"converted_transactionDate".isNull, 9999).otherwise(year($"converted_transactionDate")))
+      .withColumn("month", when($"converted_transactionDate".isNull, 99).otherwise(month($"converted_transactionDate")))
+      .withColumn("day", when($"converted_transactionDate".isNull, 99).otherwise(dayofmonth($"converted_transactionDate")))
+      .withColumn("transaction_timestamp",  when($"converted_transactionDate".isNull, to_timestamp(lit("9999-12-31 000000"), "yyyy-MM-dd HHmmss"))
+        .otherwise(to_timestamp(concat_ws(" ", $"converted_transactionDate",
+          $"applicants.primaryConsumer.transUnionUSConsumerCreditReport.transUnionUSConsumerCreditReport.transactionTime"), "yyyy-MM-dd HHmmss")))
+      .drop($"converted_transactionDate")
```
Comment (kimberlyma) [inline EquiFaxResponse_Bronze/src/main/scala/com/loanpal/etl/equifaxresponse/equifaxresponse_bronze/Main.scala]: this is a nitpick it doesn't affect the code but you could repeat what you did for **converted_transactionDate** for transactionTime and put it in its own column "converted_transactionTime" then drop it at the end so the concat is more concise and the style is consistent.

File: .idea/workspace.xml [hunk 3]
```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ChangeListManager">
-    <list default="true" id="962207ee-fee5-4773-b251-2e3c08c608bd" name="Default Changelist" comment="" />
+    <list default="true" id="962207ee-fee5-4773-b251-2e3c08c608bd" name="Default Changelist" comment="">
```
[the remark this answers is not in the document]
Comment (AnushaVedagiri) [reply .idea/workspace.xml]: Removed

File: .idea/workspace.xml (hunk 1 above)
Comment (AnushaVedagiri) [reply .idea/workspace.xml]: Removed

File: EquiFaxResponse_Bronze/src/main/scala/com/loanpal/etl/equifaxresponse/equifaxresponse_bronze/Main.scala (hunk 2 above)
Comment (AnushaVedagiri) [reply EquiFaxResponse_Bronze/src/main/scala/com/loanpal/etl/equifaxresponse/equifaxresponse_bronze/Main.scala]: Changed this

**Comment by AnushaVedagiri** [comment]
> Anusha looks like this is also missing columns, please see the table in the ticket. We will need all columns as they are described in the ticket. Specifically we are missing credit, and loan_id

**Comment by AnushaVedagiri** [comment]
Added credit and loan_id

**Comment by AnushaVedagiri** [comment]
I have already deleted .idea files in DATA-514 branch. Are you referring to the idea files in other branches?  When I merge to develop and master I think it will be deleted automatically I think. Also had .idea files added to gitignore

**Pull request merged**